ci-configs/packages-preview.json

```diff
@@ -117,9 +117,6 @@
     {
       "name": "@azure/quantum-jobs@next"
     },
-    {
-      "name": "@azure/core-xml"
```

Comment:
Why did we remove this? From what I know this is still in preview

Reply:
I am about to move it back. Just found it does not add the @next tag.